Library/Tests/ViewModels/BackingViewModelTests.swift

```diff
@@ -349,7 +349,9 @@ internal final class BackingViewModelTests: TestCase {
   }
 
   func testEstimatedDeliveryDateLabelText() {
-    let reward = .template |> Reward.lens.estimatedDeliveryOn .~ 1468527587.32843
+    let estimatedDelivery = 1468527587.32843
```

Comment:
How about we write a test that would have failed before the fix?

- It should set the environment locale to NY, and
- It should set the `estimatedDeliveryOn` to be midnight of the first of the month in UTC time.